frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -673,6 +673,14 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
             if (props.autoPlay) {
                 // Autoplay assumes we are playing immediately so lets go ahead and load more data
                 actions.setPlay()
+
+                if (router.values.searchParams.pause) {
+                    setTimeout(() => {
+                        // KLUDGE: when loaded for visual regression tests we want to pause the player
+                        // but only after it has had time to buffer and show the frame
+                        actions.setPause()
+                    }, 100)
```

Comment:
a little yucky but it _should_ let us have a test that proves we can render a recording without flapping 🤞

Reply:
Could we use `waitForSelector` in the test instead?

Reply:
Frustratingly if we start paused we never process the data so the player frame is just a black square. If we play (the default behaviour and then stop after its processed the data then we see the player screen and can assert that _at least_ the full snapshot has been processed (i.e. we didn't completely break rrweb playback)

We have to be paused so that the visual regression snapshot doesn't flap because of the seekbar timestamp changing

And don't want to be at 0 so we can see that the sekbar at least paints the "played" portion of the recording correctly

Reply:
I've added more to the comment so that the future traveller who wants to avoid this slightly yucky reaching into implementation code from tests knows what I was thinking